compiler/test/stdlib/array.test.gr

```diff
@@ -357,3 +357,12 @@ let result = Array.sort(comp, arr)
 assert arr == [> "a", "a", "ab", "abc", "abcd", "abcde"]
 // Returns void
 assert result == void
+
+
+// Array.rotate
+assert Array.rotate([> 1, 2, 3, 4, 5], 0) == [> 1, 2, 3, 4, 5]
+assert Array.rotate([> 1, 2, 3, 4, 5], 3) == [> 3, 4, 5, 1, 2]
+assert Array.rotate([> 1, 2, 3, 4, 5], -4) == [> 5, 1, 2, 3, 4]
+assert Array.rotate([> 1, 2, 3, 4, 5], 5) == [> 1, 2, 3, 4, 5]
+assert Array.rotate([> 1, 2, 3, 4, 5], 48) == [> 3, 4, 5, 1, 2]
+assert Array.rotate([> 1, 2, 3, 4, 5], -54) == [> 5, 1, 2, 3, 4]
```

Comment:
there will all need to be updated since the argument order will change and the function won't return an array.